test/manipulation.test.js

```diff
@@ -688,6 +688,313 @@ describe('manipulation', function () {
     });
   });
 
+  if (!getSchema().connector.replaceOrCreate) {
+    describe.skip('replaceOrCreate - not implemented', function(done) { done; });
```

Comment:
Also:

The callback of `describe` does not take any arguments. In fact, you don't have to provide any callback function at all.

```js
describe.skip('replaceAttributes/replaceById - not implemented');
```